README.md

```diff
@@ -156,8 +156,11 @@ config field to something that's high enough up to include both spec and source
 files, and set `srcFiles` to `[]`. You can autogenerate such a configuration by
 running `npx jasmine-browser-runner init --esm`.
 
+If you want to load ES module source directly on load instead of loading it from
+the corresponding spec, set the `modulesWithSideEffectsInSrcFiles` config property to `true`.
```

Comment:
I am unsure if I need to mention `enableTopLevelAwait` (for sequencial source loading) here, too.